src/VoyagerServiceProvider.php

```diff
@@ -117,7 +117,11 @@ protected function registerViewComposers()
      */
     protected function addStorageSymlinkAlert()
     {
-        $currentRouteAction = app('router')->current()->getAction();
+        if (app('router')->current() !== null) {
+            $currentRouteAction = app('router')->current()->getAction();
```

Comment:
But wait! @marktopper I didn't notice earlier but since that's a service provider, can't you simply `$this->app->router->current()` ?